src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -512,7 +512,7 @@ void EnGinkoMan_SetupBankAward(EnGinkoMan* this) {
 
 void EnGinkoMan_BankAward(EnGinkoMan* this, GlobalContext* globalCtx) {
     if (Actor_HasParent(&this->actor, globalCtx)) {
-        // ? when would bank have a parent?
+        // Parent is the player when starting to receive the award
         this->actor.parent = NULL;
         EnGinkoMan_SetupBankAward2(this);
     } else if (this->curTextId == 0x45B) { // "Whats this, you already saved up 200?"
```

Comment:
Here too, and anywhere else in this file.

Reply:
Wont change this or others, for reasons stated above.